tools/desktop/ignition-gazebo.desktop

```
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Igniton Gazebo
+Comment=3D Robot Simulator
+Exec=/usr/bin/ign gazebo
```

This may not work well if the user has multiple versions installed, right? How about generating it with a CMake template to inject the version like:

```suggestion
Exec=/usr/bin/ign gazebo --force-version @PROJECT_VERSION_MAJOR@
```

And then the name above could be `Ignition Gazebo Citadel`, `Ignition Gazebo Fortress`...

---

Done in [5723a33](https://github.com/ignitionrobotics/ign-gazebo/pull/1411/commits/5723a33befda36050df15df3046c4b11f1188608). I've also included the major version in the name so the user knows which major version is going to be run.